.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Android CI with Gradle
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  unit_tests:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Install NDK
+        run: echo "y" | sudo ${ANDROID_HOME}/tools/bin/sdkmanager --install "ndk;21.0.6113669" --sdk_root=${ANDROID_SDK_ROOT}
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Run unit tests with Gradle
+        run: ./gradlew jacocoTestReport --stacktrace
+      - name: Generate Javadoc with Gradle
```

Comment:
Java docs is not set up for this repo, could we remove this step?

Reply:
removed.